app/src/main/java/org/wikipedia/yearinreview/YearInReviewViewModel.kt

```diff
@@ -104,27 +112,28 @@ class YearInReviewViewModel() : ViewModel() {
             val commonsResponse = commonsCall.await()
             val wikidataResponse = wikidataCall.await()
 
-            personalizedStatistics.editCount += homeSiteResponse.query?.userInfo!!.editCount
-            personalizedStatistics.editCount += wikidataResponse.query?.userInfo!!.editCount
-            personalizedStatistics.editCount += commonsResponse.query?.userInfo!!.editCount
-
-            if (personalizedStatistics.editCount >= MINIMUM_EDIT_COUNT) {
-                editCountData.headLineText = WikipediaApp.instance.resources.getQuantityString(
-                    R.plurals.year_in_review_edit_count_headline,
-                    personalizedStatistics.editCount,
-                    personalizedStatistics.editCount
-                )
-                editCountData.bodyText = WikipediaApp.instance.resources.getQuantityString(
-                    R.plurals.year_in_review_edit_count_bodytext,
-                    personalizedStatistics.editCount,
-                    personalizedStatistics.editCount
+            val editCount = homeSiteResponse.query?.userInfo!!.editCount + wikidataResponse.query?.userInfo!!.editCount + commonsResponse.query?.userInfo!!.editCount
```

Comment:
Would it be possible to keep the existing way of adding up edit counts because they look clearer?